tests/functional/test_snap.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+"""Test exporter snap."""
+from subprocess import check_call, run
+
+SNAP_NAME = "prometheus-juju-backup-all-exporter"
+
+
+def test_default_config_installed(snap_config):
+    """Check if the snap default config is in active state."""
+    assert check_call(f"ls {snap_config}".split()) == 0
```

Comment:
Is there any reason to use subprocess here? Why not simple `os.path.exists(...)`?

Reply:
I am overthinking... yes `os.path.exists` is enough